src/interp/interp.h

```diff
@@ -503,8 +503,11 @@ class Store {
  private:
   template <typename T>
   friend class RefPtr;
+  friend class Thread;
 
   Features features_;
+  // This set contains the currently active Thread objects
+  std::set<Thread*> threads_;
```

Comment:
Maybe just make this public.. rather than added a friend?   Or add  `addThread` / `removeThread` methods (but that seems roughly the same as making it public).

Reply:
I made a `thread()` inline function to get a reference to the private field.